gradio/chat_interface.py

```diff
@@ -599,7 +602,8 @@ def _setup_events(self) -> None:
                 [self.textbox],
                 [self.textbox, self.saved_input],
                 show_api=False,
-                queue=False,
+                queue=bool(self.validator),
```

Comment:
@pngwn you had originally added the validator to the submit_fn_kwargs event but that was causing issues because the inputs of that event are state components. 

This was causing two issues:
- State components never return an output via the API. Instead they are stored in the server. That's why the output of the validation function was being passed to the user function.
- State components never display loading statuses so the error message would never be seen even if the first issue above was addressed

Instead, I'm opting to add it to the first event in the chain, which is where we store the user input. This makes a bit more sense to me as the validator only has to worry about checking the latest message (it doesn't take the entire history as output)